library/cloudformation/index.js

```diff
@@ -51,7 +56,7 @@ function CloudFormationWrapper() {
     return resp;
   }
 
-  async function deleteStack({ stackName, options={} }) {
+  async function deleteStack({ stackName, options={} } = {}) {
```

Comment:
If we are going to allow an empty object, there should be a check to make sure that `stackName` is defined.